packages/lodestar-cli/src/cmds/beacon/testnets/altona.ts

```diff
@@ -1,50 +1,34 @@
+import {IBeaconNodeOptionsPartial} from "../../../options";
+
 /* eslint-disable max-len */
 
 // Use a Typescript file instead of JSON so it's automatically included in the built files
 
-export const altonaConfig = {
-  "api": {
-    "enabled": true,
-    "namespaces": ["beacon", "validator"],
-    "host": "127.0.0.1",
-    "port": 9596,
-    "cors": {
-      "origin": "*"
-    }
-  },
-  "eth1": {
-    "enabled": false,
-    "provider": {
-      "url": "http://goerli.prylabs.net"
+export const altonaConfig: IBeaconNodeOptionsPartial = {
+  api: {
+    rest: {
```

Comment:
I have the same question here as I do in this PR:
https://github.com/ChainSafe/lodestar/pull/1267#discussion_r463761752